topics/backbonejs/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: backbone, backbonejs
+created_by: Jeremy Ashkenas
+display_name: BACKBONE.JS
```

Comment:
Most places that I see referring to Backbone.js type it as `Backbone.js` with only the `B` capitalized. Thoughts on that?

Reply:
Thank for your suggestion. This is the first time I contribute so there are many errors. I've tried adding Backbone.js to alias but that caused the error.

Reply:
My suggestion is really to just change this line to `Backbone.js` since that is what will be displayed. See here for ReactJS https://github.com/topics/reactjs, https://github.com/github/explore/blame/master/topics/react/index.md#L4.

Also, thanks for your contribution. It's people like you that make open source software great, so keep at it!

Here is the suggested change if it makes it easier:
```suggestion
display_name: Backbone.js
```


p.s. the reason that the alias of `Backbone.js` won't work is because those are used to create  alternative urls, so anything listed in the aliases list can be used:
https://github.com/topics/backbone, /backbonejs
that is the reason you can't have a period (`.`) in the name of an alias.